src/chttpd/src/chttpd_db.erl

```diff
@@ -952,21 +1025,34 @@ send_docs_multipart(Req, Results, Options1) ->
             couch_doc:doc_to_multi_part_stream(InnerBoundary, JsonBytes, Atts,
                     fun(Data) -> couch_httpd:send_chunk(Resp, Data)
                     end, true),
-             couch_httpd:send_chunk(Resp, <<"\r\n--", OuterBoundary/binary>>)
```

Comment:
This wasn't indented correctly to start with, but let's keep reformatting for a separate PR.